src/sentry/api/endpoints/organization_transaction_anomaly_detection.py

```diff
@@ -0,0 +1,104 @@
+from datetime import datetime
+
+from django.conf import settings
+from rest_framework.request import Request
+from rest_framework.response import Response
+from urllib3 import Retry, connection_from_url
+
+from sentry.api.bases import OrganizationEventsEndpointBase
+from sentry.api.utils import get_date_range_from_params
+from sentry.snuba.discover import timeseries_query
+from sentry.utils import json
+
+
+def get_anomalies(snuba_io):
+    ads_connection_pool = connection_from_url(
+        settings.ANOMALY_DETECTION_URL,
+        retries=Retry(
+            total=5,
+            status_forcelist=[408, 429, 502, 503, 504],
+        ),
+        timeout=settings.SENTRY_SNUBA_TIMEOUT,
+    )
+
+    return ads_connection_pool.urlopen(
+        "POST",
+        "/anomaly/predict",
+        body=json.dumps(snuba_io),
+        headers={"content-type": "application/json;charset=utf-8"},
+    )
+
+
+def map_snuba_queries(start, end):
```

Comment:
Just so you're aware, this function is likely going to create queries that will fail. For example
- start=today-90 days
- end=today-88 days
after this function
- start=today-95days (outside of retention)

Reply:
Yeah, start and end must stay within `[now(), now() - 90d)`. If someone is picking a relative period that far in the past it's expected that we just return what's within that window.

Reply:
@wmak I took a pass at updated the logic here in a way that avoids a `start/end` argument outside of our snuba retention window. As part of this change, I also adjusted to logic for expanding the time window. Instead of strictly going backwards from `start`, we expand our window by decreasing `start` and increasing `end` evenly. 

This would only apply to a situation where the user is trying to zoom into a specific historical time window (since by default we will be looking at a window anchored to `now` and will only be able to go backwards in time).